.github/workflows/release.yml

```diff
@@ -0,0 +1,72 @@
+name: Release Packages
+
+on:
+  pull_request:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+      dryRun:
+        type: choice
+        description: 'Dry run'
+        required: true
+        default: 'false'
+        options:
+          - 'true'
+          - 'false'
+
+env:
+  NX_NON_NATIVE_HASHER: true
+
+jobs:
+  dry-run-release:
+    # This line is critical for copy paste issues
+    if: github.repository == 'code-pushup/cli' && (github.event_name == 'pull_request' || (github.event_name == 'workflow_dispatch' && github.event.inputs.dryRun == 'true'))
+    name: Dry run release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+          cache: npm
+      - name: Install dependencies
+        run: npm ci
+      - name: Execute version target on all affected packages and release them
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: npx nx affected --base=last-release --target=version --dryRun
```

Comment:
Why does this need a `GITHUB_TOKEN` if it's a dry run? :raised_eyebrow: What does it do?

Reply:
Nice, thanks. I will remove it